packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,135 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BagInformation from './BagInformation';
+import Separator from '../separator/Separator';
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +blank: boolean,
+  +name: string,
+  +gender: 'female' | 'male' | 'other',
+  +nationality: string,
+  +dateOfBirth: string,
+  +id: string,
+  +insurance: string,
+  +bags: Array<Bag>,
+  +passengerCount: number,
+|};
+
+export default function PassengerCard(props: Props) {
+  const newPassengerTitle = `${props.passengerCount}. Passenger`;
+  const preffix =
+    props.gender === 'male' ? 'Mr.' : props.gender === 'female' ? 'Ms.' : '';
+  const passengerTitle = `${preffix} ${props.name}`;
+
+  const passenger = props.blank ? newPassengerTitle : passengerTitle;
+  const insurance = props.blank ? '' : props.insurance;
+  const dateOfBirth = props.blank ? '' : props.dateOfBirth;
+  const nationality = props.blank ? '' : props.nationality;
+  const id = props.blank ? '' : props.id;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.container_name}>
+        <Icon name="passenger" />
+        <Text size="large">{passenger}</Text>
+      </View>
+      <View style={styles.container_top}>
+        <View style={styles.row_wrapper}>
+          <Text type="secondary" style={styles.text_padding}>
+            Nationality
+          </Text>
+          <Text>{nationality}</Text>
+        </View>
```

Comment:
What about make component from this?
```
<View style={styles.row_wrapper}>
  <Text type="secondary" style={styles.text_padding}>
     Nationality
   </Text>
   <Text>{nationality}</Text>
 </View>```

Reply:
Ok! I made PassengerCardDetail component out of this